front/src/modules/companies/components/CompanyBoardCard.tsx

```diff
@@ -148,6 +193,14 @@ export const CompanyBoardCard = () => {
             pictureUrl={getLogoUrlFromDomainName(company.domainName)}
             variant={EntityChipVariant.Transparent}
           />
+          {showCompactViewOptionInCards && (
+            <StyledCompactIconContainer className="compact-icon-container">
+              <StyledIconEye
```

Comment:
I think we can use showIcon && <StyledIconEye ... /> instead of passing showIcon as a prop of the styled component